router/dataplane.go

```diff
@@ -161,6 +170,10 @@ type BatchConn interface {
 	// preconditions for IO-spec:
 	// @ requires  MsgToAbsVal(&msgs[0], egressID) == ioAbsPkts
 	// @ requires  io.token(place) && io.CBioIO_bio3s_send(place, ioAbsPkts)
+	// SIF: classification spec
+	// NOTE: I mark `ioAbsPkts` instead of `MsgToAbsVal(...)` as low here to
+	// match variable used in send permission.
```

Comment:
```suggestion
	// Classification spec:
	// NOTE: `ioAbsPkts` is marked as low instead of `MsgToAbsVal(...)` to
	// match variable used in send permission.
```